README.md

```diff
@@ -14,7 +33,9 @@ The following layers in this repository are officially supported by Mender:
 * `meta-mender-raspberrypi-demo` (Raspberry Pi 3 demo-board modifications)
 * `meta-mender-qemu` (QEMU emulator board support)
 
-The rest of the layers are community supported, and the response to issues may vary.
+Commercially available add-ons are available through the `meta-mender-commercial` layer. See the [Readme](meta-mender-commercial/README.md) for further information.
```

Comment:
Not only add-ons, but software in general,
```suggestion
Commercially available components are available through the `meta-mender-commercial` layer. See the [Readme](meta-mender-commercial/README.md) for further information.
```

Reply:
@lluiscampos sounds good!